src/figures/figure2.jl

```diff
@@ -111,7 +111,7 @@ function splot_pred(cell; Lbound = true)
 end
 
 function figure2()
-    data = loadMixData(; discard_small = true)
```

Comment:
This might break the figure. You should just filter the dataframe in this function after import.